---
title: Publications
layout: page
show_sidebar: false
---

# Publications

## Peer-reviewed papers

- **Weber CC**. 2014. Disentangling Cobionts and Contamination in Long-Read Genomic Data using Sequence Composition. G3: Genes, Genomes, Genetics ([link](https://doi.org/10.1093/g3journal/jkae187))

- **Weber CC**, Perron U, Casey D, Yang Z, Goldman N. 2021. Ambiguity coding allows accurate inference of evolutionary parameters from alignments in an aggregated state-space. Systematic Biology  ([link](https://academic.oup.com/sysbio/article/70/1/21/5827469))

- **Weber CC**, Whelan S. 2019. Physicochemical amino acid properties better describe substitution rates in large populations. Molecular Biology and Evolution ([link](https://doi.org/10.1093/molbev/msz003) ).

- Platt A*, **Weber CC** *, Liberles DA. 2018. Protein Evolution Depends On Multiple Distinct Population Size Parameters. BMC Evolutionary Biology 18 (17) ([link](https://bmcecolevol.biomedcentral.com/articles/10.1186/s12862-017-1085-x) ) *Authors contributed equally. Names in alphabetical order.

- Chi PB, Kim D, Lai JK, Bykova N, **Weber CC**, Kubelka J, Liberles DA. 2018. A new parameter‐rich structure‐aware mechanistic model for amino acid substitution during evolution. Proteins 86:2 ([link](https://onlinelibrary.wiley.com/doi/10.1002/prot.25429))

- Arenas M, **Weber CC**, Liberles DA, Bastolla U. 2017. ProtASR: An Evolutionary Framework for Ancestral Protein Reconstruction with Selection on Folding Stability. Systematic Biology 66:6, 1054-1064 ([link](https://academic.oup.com/sysbio/article/66/6/1054/2840014))

- Mugal CF, **Weber CC**, Ellegren H (2015) GC-biased gene conversion links the recombination landscape and demography to genomic base composition. BioEssays doi:10.1002/bies.201500058 ([link](https://onlinelibrary.wiley.com/doi/abs/10.1002/bies.201500058))

- **Weber CC**, Boussau B, Romiguier J, et al. 2014. Evidence for GC-biased gene conversion as a driver of between-lineage differences in avian base composition. Genome Biology 15:549 ([link](http://genomebiology.com/2014/15/12/549/)) (appeared as #1 "most forwarded" on Genome Biology's site on Dec 17th 2014)

- **Weber CC**, Nabholz B, Romiguier J, et al. 2014. Kr/Kc but not dN/dS correlates positively with body mass in birds, raising implications for inferring lineage-specific selection. Genome Biology 15:542 ([link](http://genomebiology.com/2014/15/12/542))

- Jarvis ED, Mirarab S, Aberer AJ, Li B, Houde P, Li C, Ho SYW, Faircloth BC, Nabholz B, Howard JT, Suh A, **Weber CC**, et al. 2014. Whole-genome analyses resolve early branches in the tree of life of modern birds. Science 346:1320-1331 ([link](https://www.science.org/doi/10.1126/science.1253451))

- Suh A, **Weber CC**, Kehlmaier C, et al. 2014. Early Mesozoic coexistence of amniotes and Hepadnaviridae. PLoS Genetics 10:e1004559 ([link](https://doi.org/10.1371/journal.pgen.1004559))

- **Weber CC**, Pink CJ, Hurst LD. 2012. Late-replicating domains have higher divergence and diversity in Drosophila melanogaster. Molecular Biology and Evolution 29:873–82 ([link](https://doi.org/10.1093/molbev/msr265))

- **Weber CC**, Hurst LD. 2011. Support for multiple classes of local expression clusters in Drosophila melanogaster, but no evidence for gene order conservation. Genome Biology 12:R23 ([link](http://genomebiology.com/content/12/3/R23))

- **Weber CC**, Hurst LD. 2010. Intronic AT Skew is a Defendable Proxy for Germline Transcription but does not Predict Crossing-Over or Protein Evolution Rates in Drosophila melanogaster. Journal of Molecular Evolution: 415–426 ([link](https://link.springer.com/article/10.1007/s00239-010-9395-2))

- **Weber CC**, Hurst LD. 2009. Protein rates of evolution are predicted by double-strand break events, independent of crossing-over rates. Genome Biology and Evolution 1:340–9 ([link](https://doi.org/10.1093/gbe/evp033))

- Warnecke T, **Weber CC**, Hurst LD. 2009. Why there is more to protein evolution than protein function: splicing, nucleosomes and dual-coding sequence. Biochemical Society transactions 37:756–61 ([link](https://doi.org/10.1042/BST0370756))


## Data notes:
- Jarvis ED, Mirarab S, Aberer AJ, Li B, Houde P, Li C, Ho SYW, Faircloth BC, Nabholz B, Howard JT, Suh A, **Weber CC**, et al. Phylogenomic Analyses Data of the Avian Phylogenomics Project. GigaScience ([link](https://doi.org/10.1186/s13742-014-0038-1))

## Selected genome notes:
Some genomes I looked at prior to their release (I am listed as a group author)

- Lohse et al. 2022. The genome sequence of the Adonis blue, Lysandra bellargus (Rottemburg, 1775). Wellcome Open Research. ([link](https://wellcomeopenresearch.org/articles/7-255))
- Ebdon et al. 2021. The Genome Sequence of the Clouded Yellow, Colias crocea (Geoffroy, 1785). Wellcome Open Research.
- Lohse et al. 2021. The Genome Sequence of the Large White, Pieris Brassicae (Linnaeus, 1758). Wellcome Open Research

## PhD Thesis:
- **Weber, CC**. 2011. Genome structure and determinants of rates of evolution. Thesis (Doctor of Philosophy (PhD)). University of Bath ([link](https://researchportal.bath.ac.uk/en/studentTheses/genome-structure-and-determinants-of-rates-of-evolution))
